src/Integration.Vsix/Manifests/VS2022/source.extension.vsixmanifest

```diff
@@ -14,7 +14,7 @@
         <Tags>SonarLint;SonarQube;Analysis;Roslyn;CodeAnalysis;Analyzer;Code analysis;Sonar;Debt;Technical;Tech;Quality</Tags>
     </Metadata>
     <Installation>
-      <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+      <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0.31808.319,18.0)">
```

Comment:
Limit the installation to RC candidates (or the equivalent Preview build, since the version numbers are the same): https://docs.microsoft.com/en-us/visualstudio/install/visual-studio-build-numbers-and-release-dates?view=vs-2022